packages/integration-tests/package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Test repository to run integration tests",
   "scripts": {
     "start": "concurrently \"yarn start-client\" \"yarn start-api\"",
-    "start-client": "BROWSER=none REACT_APP_START_DATE=2020-01-01 REACT_APP_END_DATE=2021-03-01 REACT_APP_GROUP_BY=month REACT_APP_PAGE_LIMIT=1000 yarn workspace @cloud-carbon-footprint/client start",
-    "start-api": "TEST_MODE=true CACHE_MODE=LOCAL yarn workspace @cloud-carbon-footprint/api start",
+    "start-client": "BROWSER=none REACT_APP_START_DATE=2020-01-01 REACT_APP_END_DATE=2021-03-01 REACT_APP_GROUP_BY=month REACT_APP_PAGE_LIMIT=1000 REACT_APP_BASE_URL=http://127.0.0.1:3000/api yarn workspace @cloud-carbon-footprint/client start",
```

Comment:
Sure - I found that If I had a custom REACT_APP_BASE_URL and ENABLE_CORS vars set in my local environment then it prevented the integration tests from running, since they use a hardcoded URL in the tests. By specifying REACT_APP_BASE_URL to be the one used in the tests, and ensuring CORS is disabled, it prevents any problems